### PR TITLE
Chinese localization

### PR DESCRIPTION
**Pull request by auroursa**
I am a native Chinese speaker, and this 'messages.po' is my translation based on the English template of 1.67. So there should be work.
* Create messages.po in Chinese (Simplified).
* Update languages.ts

### CODE REVIEW

File: src/locale/languages.ts [hunk 1]
```diff
@@ -35,6 +36,7 @@ export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.pt_BR, name: 'Português (BR) – Portuguese (BR)'},
   {code2: AppLanguage.uk, name: 'Українська – Ukrainian'},
   {code2: AppLanguage.ca, name: 'Català – Catalan'},
+  {code2: AppLanguage.ja, name: '简体中文（中国） – Chinese (Simplified)'},
```
Comment (surfdude29) [inline src/locale/languages.ts:39]: I think it should be AppLanguage.zh_CN here

Comment (auroursa) [reply src/locale/languages.ts]: Thanks for the review, I've fixed the issue and cross-checked the translation with my friend.

File: src/locale/locales/zh-CN/messages.po [hunk 2]
```diff
@@ -0,0 +1,4230 @@
...
+
+#: src/view/com/profile/ProfileHeader.tsx:632
+msgid "{following} following"
+msgstr "{following} 正在关注"
+
+#: src/view/shell/desktop/RightNav.tsx:151
+msgid "{invitesAvailable, plural, one {Invite codes: # available} other {Invite codes: # available}}"
+msgstr "{invitesAvailable, plural, one {邀请码: # 可用} other {邀请码: # 可用}}"
+
+#: src/view/screens/Settings.tsx:435 src/view/shell/Drawer.tsx:664
```
Comment (surfdude29) [inline src/locale/locales/zh-CN/messages.po:32]: One other thing I noticed that you might want to check: all the other messages.po files seem to have each code reference on a separate line, and I'm not sure but I think that might be the format expected by Lingui? So it's possible that you might need to add some line breaks, like here, for instance

Comment (auroursa) [reply src/locale/locales/zh-CN/messages.po]: I referenced the format of other "messages.po" files and added extra line breaks between strings. sorry for any oversight on my part.

Comment (surfdude29) [reply src/locale/locales/zh-CN/messages.po]: No need to apologise, it's just that I had noticed other contributors who had used Poedit had run into that issue before with the line breaks, so I wanted to mention it, that's all :)

**Comment by RitsukiP** [comment]
A native Chinese speaker. I made a little contribution about revising translating accuracy and giving advice to @auroursa .
我以中文为母语，在改善翻译质量和向 @auroursa  提出建议方面做出了一点微小的贡献

File: src/locale/locales/zh-CN/messages.po [hunk 3]
```diff
@@ -0,0 +1,4345 @@
...
+msgid "Account muted"
+msgstr "已静音账户"
+
+#: src/view/com/modals/ModerationDetails.tsx:86
+msgid "Account Muted"
+msgstr "已静音账户"
+
+#: src/view/com/modals/ModerationDetails.tsx:72
+msgid "Account Muted by List"
+msgstr "来自列表的已静音账户"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:111]: The sentence is very strange

File: src/locale/locales/zh-CN/messages.po [hunk 4]
```diff
@@ -0,0 +1,4345 @@
...
+msgstr "添加App专用密码"
+
+#: src/view/com/modals/report/InputIssueDetails.tsx:41
+#: src/view/com/modals/report/Modal.tsx:191
+msgid "Add details"
+msgstr "添加细节"
+
+#: src/view/com/modals/report/Modal.tsx:194
+msgid "Add details to report"
+msgstr "在反馈中添加细节"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:168]: "补充反馈详细内容"

File: src/locale/locales/zh-CN/messages.po (hunk 3 above)
Comment (auroursa) [reply src/locale/locales/zh-CN/messages.po:111]: I checked the documentation and it seems that it should translate to "账户已被列表静音"

File: src/locale/locales/zh-CN/messages.po [hunk 5]
```diff
@@ -0,0 +1,4345 @@
...
+msgstr "创建带有缩略图的卡片。该卡片链接到 {url}"
+
+#: src/view/com/modals/ChangeHandle.tsx:389
+#: src/view/com/modals/ServerInput.tsx:102
+msgid "Custom domain"
+msgstr "自定义域名"
+
+#: src/view/screens/PreferencesExternalEmbeds.tsx:55
+msgid "Customize media from external sites."
+msgstr "自定义来自外部站点的媒体。"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:918]: "自定义外部站点的媒体。"

File: src/locale/locales/zh-CN/messages.po (hunk 4 above)
Comment (auroursa) [reply src/locale/locales/zh-CN/messages.po:168]: I will fix it.

File: src/locale/locales/zh-CN/messages.po [hunk 6]
```diff
@@ -0,0 +1,4345 @@
...
+msgid "Enable this setting to only see replies between people you follow."
+msgstr "启用此设置以仅查看你关注的人之间的回复。"
+
+#: src/view/screens/Profile.tsx:427
+msgid "End of feed"
+msgstr "结束信息源"
+
+#: src/view/com/modals/AddAppPasswords.tsx:165
+msgid "Enter a name for this App Password"
+msgstr "为此App专用密码命名"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:1209]: "为此 App 专用密码命名"

File: src/locale/locales/zh-CN/messages.po [hunk 7]
```diff
@@ -0,0 +1,4345 @@
...
+msgid "Here are some popular topical feeds. You can choose to follow as many as you like."
+msgstr "这里是一些流行的信息源供你挑选。"
+
+#: src/screens/Onboarding/StepTopicalFeeds.tsx:72
+msgid "Here are some topical feeds based on your interests: {interestsText}. You can choose to follow as many as you like."
+msgstr "这里是一些基于你兴趣所推荐的信息源供你挑选：{interestsText}。"
+
+#: src/view/com/modals/AddAppPasswords.tsx:152
+msgid "Here is your app password."
+msgstr "这里是你的App专用密码。"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:1568]: "这里是你的 App 专用密码。"

File: src/locale/locales/zh-CN/messages.po [hunk 8]
```diff
@@ -0,0 +1,4345 @@
...
+msgid "Input code sent to your email for password reset"
+msgstr "输入发送到你电子邮箱的验证码以重置密码"
+
+#: src/view/com/modals/DeleteAccount.tsx:180
+msgid "Input confirmation code for account deletion"
+msgstr "输入删除用户的验证码"
+
+#: src/view/com/auth/create/Step1.tsx:144
+msgid "Input email for Bluesky account"
+msgstr "输入Bluesky账户的电子邮箱"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:1690]: "输入 Bluesky 账户的电子邮箱"

File: src/locale/locales/zh-CN/messages.po [hunk 9]
```diff
@@ -0,0 +1,4345 @@
...
+msgid "Input the username or email address you used at signup"
+msgstr "输入注册时使用的用户名或电子邮箱"
+
+#: src/view/com/auth/create/Step2.tsx:270
+msgid "Input the verification code we have texted to you"
+msgstr "输入收到的短信验证码"
+
+#: src/view/com/modals/Waitlist.tsx:90
+msgid "Input your email to get on the Bluesky waitlist"
+msgstr "输入你的电子邮箱以加入Bluesky候补列表"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:1726]: "输入你的电子邮箱以加入 Bluesky 候补列表"

File: src/locale/locales/zh-CN/messages.po [hunk 10]
```diff
@@ -0,0 +1,4345 @@
...
+msgid "Learn more about what is public on Bluesky."
+msgstr "了解有关Bluesky公开内容的更多详情。"
+
+#: src/view/com/modals/lang-settings/ContentLanguagesSettings.tsx:82
+msgid "Leave them all unchecked to see any language."
+msgstr "全部不选中以查看任何语言。"
+
+#: src/view/com/modals/LinkWarning.tsx:51
+msgid "Leaving Bluesky"
+msgstr "离开Bluesky"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:1844]: "离开 Bluesky"

File: src/locale/locales/zh-CN/messages.po [hunk 11]
```diff
@@ -0,0 +1,4345 @@
...
+#: src/view/com/util/moderation/PostAlerts.tsx:40
+#: src/view/com/util/moderation/PostHider.tsx:78
+#: src/view/com/util/moderation/ProfileHeaderAlerts.tsx:49
+#: src/view/com/util/moderation/ScreenHider.tsx:101
+msgid "Learn more about this warning"
+msgstr "了解关于这个警告的更多详情"
+
+#: src/view/screens/Moderation.tsx:242
+msgid "Learn more about what is public on Bluesky."
+msgstr "了解有关Bluesky公开内容的更多详情。"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:1836]: "了解更多关于 Bluesky 的公开内容"

File: src/locale/locales/zh-CN/messages.po [hunk 12]
```diff
@@ -0,0 +1,4345 @@
...
+msgstr "未找到"
+
+#: src/view/com/modals/VerifyEmail.tsx:246
+#: src/view/com/modals/VerifyEmail.tsx:252
+msgid "Not right now"
+msgstr "不是现在"
+
+#: src/view/screens/Moderation.tsx:232
+msgid "Note: Bluesky is an open and public network. This setting only limits the visibility of your content on the Bluesky app and website, and other apps may not respect this setting. Your content may still be shown to logged-out users by other apps and websites."
+msgstr "注意：Bluesky是一个开放的公共网络。此设置项仅限制你的内容在Bluesky应用和网站上的可见性，其他应用可能不尊从此设置项，仍可能会向未登录的用户显示你的动态。"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:2310]: "注意：Bluesky 是一个开放的公共网络。此设置项仅限制你的内容在 Bluesky 应用和网站上的可见性，其他应用可能不尊从此设置项，仍可能会向未登录的用户显示你的动态。"

File: src/locale/locales/zh-CN/messages.po [hunk 13]
```diff
@@ -0,0 +1,4345 @@
...
+msgid "Select your phone's country"
+msgstr "选择你的电话区号"
+
+#: src/view/screens/LanguageSettings.tsx:190
+msgid "Select your preferred language for translations in your feed."
+msgstr "选择你在订阅信息源中希望进行翻译的目标首选语言。"
+
+#: src/screens/Onboarding/StepAlgoFeeds/index.tsx:114
+msgid "Select your primary algorithmic feeds"
+msgstr "选择你的主要算法信息源"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:3146]: "选择你的信息源主要算法"

File: src/locale/locales/zh-CN/messages.po [hunk 14]
```diff
@@ -0,0 +1,4345 @@
...
+
+#: src/screens/Onboarding/StepAlgoFeeds/index.tsx:130
+msgid "Select your secondary algorithmic feeds"
+msgstr "选择你的次要算法信息源"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:3150]: "选择你的信息源次要算法"

File: src/locale/locales/zh-CN/messages.po [hunk 15]
```diff
@@ -0,0 +1,4345 @@
...
+msgid "Unfollow"
+msgstr "取关"
+
+#: src/view/com/profile/ProfileHeader.tsx:524
+msgid "Unfollow {0}"
+msgstr "取关 {0}"
+
+#: src/view/com/auth/create/state.ts:300
+msgid "Unfortunately, you do not meet the requirements to create an account."
+msgstr "不幸的是，你不符合创建账户的要求。"
```
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:3868]: "很遗憾，你不符合创建账户的要求。"

File: src/locale/locales/zh-CN/messages.po
Comment (U2FsdGVkX1) [inline src/locale/locales/zh-CN/messages.po:1]: "信息源" -> "信息流"

**Review by U2FsdGVkX1 — COMMENTED** [review]
Using "信息流" instead of "信息源"

**Comment by auroursa** [comment]
@U2FsdGVkX1 It looks good

**Review by auroursa — COMMENTED** [review]
I assume it's ready to go.

**Comment by auroursa** [comment]
I noticed that there was a change in upstream, so I translated the Chinese text based on the latest messages.po.
Would it be possible to help review this, I would appreciate it. @surfdude29 
Thank you, have a good day.

File: src/locale/locales/zh-CN/messages.po [hunk 16]
```diff
@@ -0,0 +1,4449 @@
...
+msgid "Help"
+msgstr "帮助"
+
+#: src/screens/Onboarding/StepSuggestedAccounts/index.tsx:132
+msgid "Here are some accounts for you to follow"
+msgstr "这里是一些推荐关注的用户"
+
+#: src/screens/Onboarding/StepSuggestedAccounts/index.tsx:132
+#~ msgid "Here are some accounts for your to follow"
+#~ msgstr ""
```
Comment (surfdude29) [inline src/locale/locales/zh-CN/messages.po:1608]: In case you want to remove them, also here

File: src/locale/locales/zh-CN/messages.po [hunk 17]
```diff
@@ -0,0 +1,4449 @@
...
+msgid "Liked by {likeCount} {0}"
+msgstr "{likeCount} {0} 点赞"
+
+#: src/view/com/notifications/FeedItem.tsx:170
+msgid "liked your custom feed"
+msgstr "点赞你的自定义信息流"
+
+#: src/view/com/notifications/FeedItem.tsx:171
+#~ msgid "liked your custom feed '{0}'"
+#~ msgstr "点赞你的自定义信息流 '{0}'"
```
Comment (surfdude29) [inline src/locale/locales/zh-CN/messages.po:1959]: Here

File: src/locale/locales/zh-CN/messages.po [hunk 18]
```diff
@@ -0,0 +1,4449 @@
...
+msgid "Option {0} of {numItems}"
+msgstr "第 {0} 个选项，共 {numItems} 个"
+
+#: src/view/com/modals/Threadgate.tsx:89
+msgid "Or combine these options:"
+msgstr "或者选择组合这些选项："
+
+#: src/screens/Onboarding/StepAlgoFeeds/index.tsx:122
+#~ msgid "Or you can try our \"Discover\" algorithm:"
+#~ msgstr "或者你可以尝试我们的“发现”算法："
```
Comment (surfdude29) [inline src/locale/locales/zh-CN/messages.po:2553]: And here

File: src/locale/locales/zh-CN/messages.po [hunk 19]
```diff
@@ -0,0 +1,4449 @@
...
+msgid "There's something wrong with this number. Please choose your country and enter your full phone number!"
+msgstr "电话号码有误，请选择电话区号并输入完整的电话号码！"
+
+#: src/screens/Onboarding/StepSuggestedAccounts/index.tsx:138
+msgid "These are popular accounts you might like:"
+msgstr "这里是一些受欢迎的账号，你可能会喜欢："
+
+#: src/screens/Onboarding/StepSuggestedAccounts/index.tsx:138
+#~ msgid "These are popular accounts you might like."
+#~ msgstr "这里是一些受欢迎的账号，你可能会喜欢。"
```
Comment (surfdude29) [inline src/locale/locales/zh-CN/messages.po:3800]: And here

**Comment by surfdude29** [comment]
As far as I can tell, it looks great on a technical level 👍

Sorry I can't review the actual new translations, but hopefully one of the other native Chinese speakers who helped before can have another look just to double-check.

Have a good day yourself :)

**Review by U2FsdGVkX1 — COMMENTED** [review]
I've checked and I think the new translations seems okay

**Review by pfrazee — APPROVED** [review]
Wonderful! Thank you everyone!

I'm seeing multiple reviews by community members and that's enough to give this the 👍

**Pull request merged**